### PR TITLE
Add compliance workflow for PCO Prod3.

### PR DESCRIPTION
**Pull request by garloff**
Using an application credential, because it's more elegant.

### CODE REVIEW

**Comment by garloff** [comment]
The second half to address #371 ...

File: .github/scs-compliance-check/openstack/clouds.yaml [hunk 1]
```diff
@@ -29,6 +29,14 @@ clouds:
     region_name: "prod2"
     interface: "public"
     identity_api_version: 3
+  pco-prod3:
+    auth_type: "v3applicationcredential"
+    auth:
+      auth_url: https://prod3.api.pco.get-cloud.io:5000
+      application_credential_id: "fe66c4c8cd3b4ea08262424783e1c58f"
+    region_name: "prod2"
```
Comment (mbuechse) [inline .github/scs-compliance-check/openstack/clouds.yaml:37]: Is that correct?

**Review by mbuechse — COMMENTED** [review]
One question that should be answered, apart from that, LGTM

File: .github/scs-compliance-check/openstack/clouds.yaml (hunk 1 above)
Comment (garloff) [reply .github/scs-compliance-check/openstack/clouds.yaml:37]: No.
Thanks for spotting.

**Review by mbuechse — APPROVED** [review]

**Pull request merged**